### DISCUSSION TITLE
Quick search sites with querys like brave

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [qutebrowser/qutebrowser#6748] [Ideas] [answered]
https://user-images.githubusercontent.com/61395246/137620414-0145cb93-7d16-47e6-b3e5-fe8a6a94063a.mp4

Do we have anything like this in QuteBrowser?
If not, can we get it as a feature maybe ? 
I'm pretty sure i could write a hacky script with rofi but i want it to be impeded , is there a way to make it so , by lets say , writing a python script and embedding it trough some API?

**Comment by ottopiramuthu** [comment, marked as answer]
You can change the `url.searchengines` [setting](https://qutebrowser.org/doc/help/settings.html#url.searchengines) to include more search engines like `'y' : 'https://www.youtube.com/results?search_query={}'`. Then opening a page like `:open y qutebrowser` (by typing `oy qutebrowser<Enter>`) will open the YouTube search for qutebrowser.
